tests/integration/cloudformation/test_cloudformation_events.py

```diff
@@ -215,3 +215,29 @@ def test_event_rule_to_logs(
     finally:
         cleanup_changesets([change_set_id])
         cleanup_stacks([stack_id])
+
+
+def test_event_rule_creation_without_target(
```

Comment:
We've moved a lot of tests already to the newer `deploy_cfn_template` fixture which simplifies this setup here greatly

The usage would be something like: 
```python
deploy_cfn_template(template_path=os.path.join(......), parameters={"EventRuleName": event_rule_name}) # see now below re: parameters
```

The fixture will also automatically clean up the stack and any created change sets.